SPECS/lttng-tools/lttng-tools.spec

```diff
@@ -16,7 +16,7 @@ BuildRequires: elfutils-devel
 BuildRequires: popt-devel
 BuildRequires: userspace-rcu-devel >= 0.8.0
 BuildRequires: lttng-ust-devel >= 2.9.0
-Requires:      lttng-ust = %{version}
+Requires:      lttng-ust >= 2.13.1
 Requires:      userspace-rcu
 Requires:      elfutils
 Requires:      nss
```

Comment:
Small ask: Would you mind checking to see if the NSS requirement is still needed, and if it can be removed/substituted with OpenSSL?